### ISSUE TITLE
The animated GIF is wrong

### ISSUE THREAD

**Issue by antocuni** [pyscript/pyscript-cli#31]
The animated GIF in the README shows an user typing `pip install pyscript-cli`, which is now outdated.

**Comment by mattkram** [comment]
Good catch! I believe I've got a script that can recreate it and I'd love it to be automated. Will look into it and at a minimum regenerate locally.

If anyone knows a good way to mock typing and generating a CLI GIF, I'm all ears! I've tried a few things in the past and never got fully where I wanted to be.

**Comment by antocuni** [comment]
Ah, I thought you already had a working solution for that. A quick googling revealed this:
https://www.terminalgif.com/

although it would be nice to have a more automated way

**Comment by marimeireles** [comment]
You can use quicktimeplayer for that... I guess?
I'm not sure if I had to use ffmpeg later to convert it to gif. But it comes in your mac and films your screen! :)
